CODEOWNERS

```diff
@@ -0,0 +1,21 @@
+# Code Ownership for buzzmobile.
+# Order matters. The last match is used, so more specific owners should be 
+# last.
+
+# Default owners/reviewers for anything.
+* @irapha @joshuamorton
+
+# Language specific reviewers:
+*.py @iraphael @joshuamorton
+*.cpp @chsahit
```

Comment:
Does this mean that sahit is the only one who can approve `*.cpp` PRs, even though like 6 says iraphael and joshuamorton both can approve "`*`?